app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -111,7 +111,7 @@ $include-html-global-classes: $include-html-classes;
 // $steel       : #666666;
 // $charcoal    : #555555;
 // $tuatara     : #444444;
-// $oil         : #333333;
+ $oil         : #7E8F7C;
```

Comment:
Color `#7E8F7C` should be written as `#7e8f7c`<br>Line should be indented 0 spaces, but was indented 1 spaces